src/group_theory/perm/sign.lean

```diff
@@ -761,3 +765,57 @@ end congr
 end sign
 
 end equiv.perm
+
+section alternating_subgroup
+open equiv.perm
+variables (α) [fintype α] [decidable_eq α]
+
+/-- The alternating group on a finite type, realized as a subgroup of `equiv.perm`.
+  For $A_n$, use `alternating_subgroup (fin n)`. -/
+def alternating_subgroup : subgroup (perm α) :=
+sign.ker
+
+variables {α}
+
+lemma alternating_subgroup_eq_sign_ker : alternating_subgroup α = sign.ker := rfl
+
+@[simp]
+lemma mem_alternating_subgroup {f : perm α} :
+  f ∈ alternating_subgroup α ↔ sign f = 1 :=
+monoid_hom.mem_ker _
+
+instance alternating_subgroup.fintype :
+  fintype (alternating_subgroup α) :=
+fintype.subtype (univ.filter (λ x, sign x = 1)) (λ x, begin
+  simp only [true_and, mem_filter, mem_univ, ← mem_alternating_subgroup],
+  refl,
+end)
+
+lemma two_mul_card_alternating_subgroup [nontrivial α] :
+  2 * card (alternating_subgroup α) = (card α).factorial :=
```

Comment:
This might be a more useful lemma if it did less work, and had `card (perm \a)` on the LHS

Reply:
Are you looking for
`card (perm α) = 2 * card (alternating_subgroup α)`?

Reply:
Sorry. I meant RHS

Reply:
It'd be nice if the index of subgroups was defined...